modules/configuring-custom-prometheus-metrics.adoc

```diff
@@ -0,0 +1,70 @@
+//module included in the following assemblies:
+// * configuration/monitor-acs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-custom-prometheus-metrics_{context}"]
+= Configuring custom Prometheus metrics
+
+[role="_abstract"]
+Use the API to configure custom Prometheus metrics for {product-title-short}. The metrics configuration is a part of the private config section of the `/v1/config` service request or response payload.
+
+.Procedure
+
+. Get the current configuration by running the following command:
++
+[source,terminal]
+----
+$ curl $ROX_API_ENDPOINT/v1/config -H "$ROX_AUTH" | jq
+{
+  "publicConfig": { ... }
+  "privateConfig": { ... }
+}
+----
+. Add some metrics to the configuration. For example, run the following command to add some image vulnerability data, including namespace and deployment data:
++
+[source,terminal]
+----
+$ curl -X PUT "$ROX_API_ENDPOINT/v1/config" -H "$ROX_AUTH" --data='
+{
+  "config": {
+    "publicConfig": { ... }
+    "privateConfig": {
+      ...
+      "metrics": {
+        "imageVulnerabilities": {
+          "gatheringPeriodMinutes": 10,
+          "descriptors": {
+            "deployment_severity": {
+              "labels": [
+                "Cluster",
+                "Namespace",
+                "Deployment",
+                "IsPlatformWorkload",
+                "IsFixable",
+                "Severity"
+              ]
+            },
+            "namespace_severity": {
+              "labels": [
+                "Cluster",
+                "Namespace",
+                "Severity"
+              ]
+            }
+          }
+        }
+...
+}
+'
+----
+. You can edit the configuration content with a single command, as shown in the following example:
```

Comment:
This appears as point 3, while it could be done instead of point 2.

I don't know our preferences in how to guide users with some magic terminal commands. The `jq` script below might not be obvious to the users.